lib/rutema/core/engine.rb

```diff
@@ -169,12 +270,25 @@ def initialize queue,configuration
       @streaming_reporters<<@collector
       @configuration=configuration
     end
-    #Call this to establish a queue with the given identifier
+
+    ##
+    # Call this to establish a queue with the given identifier
+    #
+    # This method will create a new queue within the Dispatcher instance into
+    # which data from the incoming queue from the Engine instance will be
+    # demultiplexed too.
+    #
+    # * +identifier+ - a unique identifier for the queue. If two identifiers
+    #   collide the later added subscriber will take the place of the earlier
```

Comment:
the new subscriber will replace the earlier one

Reply:
Changed :+1: